src/main/java/seedu/address/logic/parser/UpdateSessionRemarkCommandParser.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SESSION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SESSION_REMARK;
+
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.UpdateSessionRemarkCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.session.SessionNumber;
+import seedu.address.model.session.SessionRemark;
+
+/**
+ * Parses input arguments and creates a new UpdateSessionRemarkCommand object.
+ */
+public class UpdateSessionRemarkCommandParser implements Parser<UpdateSessionRemarkCommand> {
+
+    /**
+     * Parses the given `args` and creates a new `UpdateSessionRemarkCommand` object.
+     *
+     * @param args The input arguments provided by the user.
+     * @return A `UpdateSessionRemarkCommand` object.
+     * @throws ParseException If the user input does not conform to the expected format.
+     */
+    public UpdateSessionRemarkCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_SESSION, PREFIX_SESSION_REMARK);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_SESSION, PREFIX_SESSION_REMARK)
+                || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(
+                    MESSAGE_INVALID_COMMAND_FORMAT, UpdateSessionRemarkCommand.MESSAGE_USAGE));
+        }
+
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_SESSION);
```

Comment:
iirc, you can merge these 2 together:
`argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_SESSION, PREFIX_SESSION_REMARK);`

Reply:
Thanks!